# High_dimensional_data_analyses

Evaluation in python based in the article:
Wang F., Mukherjee, S., Richardson, S. and Hill, S. M. (2020) High-dimensional regression in practice:
an empirical study of finite-sample prediction, variable selection and ranking. Statistics and
Computing 30, 697â€“719. DOI: https://doi.org/10.1007/s11222-019-09914-9

For the semisynthetic data simulations, we used gene expression data from The Cancer Genome Atlas (TCGA) ovarian cancer study as covariates. In particular, we used a dataset as provided in the Supplementary Appendix of Tucker et al. (2014): an RData file called tcgaExpression.RData which is contained in a zip file called rDataObjects.zip. This zip file can be downloaded from https://bioinformatics.mdanderson.org/Supplements/ResidualDisease. It's available in the repository from the real study GitHub:
https://github.com/fw307/high_dimensional_regression_comparison.

All the article is made in R, the aim of this project was to reproduce in Python, compare results, made further analyses and provide the codes in more frequent programming language.

It's used the Jupyter Notebook environment, available by Anaconda. 


REQUIREMENTS:
numpy 
seaborn 
matplotlib
pandas
sklearn
pyreadr

